### ISSUE TITLE
[crit] 619#0: *39975386 SSL_do_handshake() failed (SSL: error:140A1175:SSL routines:SSL_BYTES_TO_CIPHER_LIST:inappropriate fallback) while SSL handshaking,

### ISSUE THREAD

**Issue by xiukongtiao** [alibaba/tengine#780]
error.log里经常报这个错误：
[crit] 619#0: *39975386 SSL_do_handshake() failed (SSL: error:140A1175:SSL routines:SSL_BYTES_TO_CIPHER_LIST:inappropriate fallback) while SSL handshaking,
请问，能解释这是出了什么问题？要如何解决，谢谢！

**Comment by chobits** [comment]
see here: http://stackoverflow.com/questions/28010492/nginx-critical-error-with-ssl-handshaking

> This looks like someone checking if the server supports TLS_FALLBACK_SCSV, which it does in your case. Nothing to worry about. On the contrary this means that your server supports a useful security feature.

**Comment by xiukongtiao** [comment]
Thank you!
someone? Do you mean someone do some checking, they are not normal user, right?
but we have checked the access log, all of the users are very normal user.
someone is using chrome,  someone is using firefox, someone using IE.
all of them looks very normal,  we do not think they did some checking?

**Comment by xiukongtiao** [comment]
I want to know, if the server can retuan a correct response when we see this error?
If this situation impact the user access our webservice?
How to reproduce this situation? How to fix and avoid that?

I am worried, our user can not access our website when we see this error.

**Comment by chobits** [comment]
> but we have checked the access log, all of the users are very normal user.
> someone is using chrome, someone is using firefox, someone using IE.
> all of them looks very normal, we do not think they did some checking?
You cannot see unnormal traffic in access.log, only error.log can log it.
And for ssl-handshaking error, nginx will not log it into access.

**Comment by chobits** [comment]
> I want to know, if the server can retuan a correct response when we see this error?

No, SSL handshake has failed already, then nginx closed this connection before receive request.
And nginx cannot receive & handle request on corrupt SSL stream.

> If this situation impact the user access our webservice?

I think it does not impact ur user normal accessing. Normal client (chrome/firefox/ie...) will not trigger this error reporting during SSL handshake

**Comment by LZTlzt** [comment]
I have the same problem now. How did you solve it
